timemachine/fe/rest/queries.py

```diff
@@ -0,0 +1,29 @@
+from rdkit import Chem
+from rdkit.Chem import rdMolDescriptors
+
+from .bond import CanonicalBond, mkbond
+
+
+def get_rotatable_bonds(mol: Chem.rdchem.Mol) -> set[CanonicalBond]:
+    """
+    Identify rotatable bonds in a molecule.
+
+    NOTE: This is an extremely crude and inaccurate method. This misses simple cases like benzoic acids, amides, etc.
+
+    Parameters
+    ----------
+    mol: ROMol
+        Input molecule
+
+    Returns
+    -------
+    set of CanonicalBond
+        Set of bonds identified as rotatable
+    """
+    pattern = Chem.MolFromSmarts("[!$(*#*)&!D1]-&!@[!$(*#*)&!D1]")
```

Comment:
OpenEye can find rotatable bonds (https://docs.eyesopen.com/toolkits/python/oechemtk/OEChemClasses/OEBondBase.html#OEChem::OEBondBase::IsRotor), may be worth investigating later